paasta_tools/cleanup_kubernetes_cr.py

```diff
@@ -105,11 +106,16 @@ def cleanup_all_custom_resources(
         )
         if not config_dicts:
             continue
+        if crd.file_prefix in INSTANCE_TYPE_TO_NAMESPACE_LOADER:
+            namespace = INSTANCE_TYPE_TO_NAMESPACE_LOADER[crd.file_prefix]
+        else:
+            namespace = ""
```

Comment:
can the default be None to avoid confusion? i had to trace the code a bit to figure out if this was safe or not since i'm 90% sure that an empty string is not a valid namespace name

maybe something like
```suggestion
        # if there's no explicit mapping from instance type to a namespace,
        # the None value here will serve as a sentinel later on for the "real"
        # default value (which is based on the CR kind)
        namespace = INSTANCE_TYPE_TO_NAMESPACE_LOADER.get(crd.file_prefix, None)
```